protostar-rust/tests/e2e/common/runner.rs

```diff
@@ -0,0 +1,17 @@
+use anyhow::{Context, Result};
+use assert_fs::fixture::PathCopy;
+use camino::Utf8PathBuf;
+use snapbox::cmd::{cargo_bin, Command as SnapboxCommand};
+
+pub fn runner() -> SnapboxCommand {
+    let snapbox = SnapboxCommand::new(cargo_bin!("rust_test_runner"));
+    snapbox
+}
+
+pub fn corelib_path() -> String {
+    let corelib = Utf8PathBuf::from("../../cairo/corelib/src")
+        .canonicalize_utf8()
+        .expect("Failed to find corelib")
+        .to_string();
+    corelib
+}
```

Comment:
Hint: you could use https://docs.rs/once_cell/latest/once_cell/sync/struct.Lazy.html as a static variable (could be inside this function) to permanently store the String and thus be able to return `&'static str` from this function. What would lead to much more pleasable usage of this function in tests